autha/src/router/create.rs

```diff
@@ -35,6 +35,7 @@ lazy_static! {
     .unwrap();
     pub static ref BIRTH: Regex =
         Regex::new(r"^\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])$").unwrap();
+    pub static ref LOCALE: Regex = Regex::new(r"^[a-z]{2}$").unwrap();
```

Comment:
I believe preciseness would be better by having a table with ISO 639-1 languages and testing inputs against it, instead of using a simple regular expression.  For instance, `zz` is not a valid ISO 639-1 code.

Consider using a self-made table of codes, or use [`isolang`](https://crates.io/crates/isolang), which is efficient and complete.